src/components/Shared/DecisionResult.tsx

```diff
@@ -2,6 +2,7 @@ import React from "react";
 import LegislationAdoptedIcon from "../Cards/LegislationCard/LegislationAdoptedIcon";
 import LegislationRejectedIcon from "../Cards/LegislationCard/LegislationRejectedIcon";
 import LegislationInProgressIcon from "../Cards/LegislationCard/LegislationInProgressIcon";
```

Comment:
I know it's not directly a part of this pr but could you move these icons to the "Shared" directory as well?